source/install/install-rhel-8-mysql.rst

```diff
@@ -11,6 +11,12 @@ Install and set up the database for use by the Mattermost server. You can instal
 
 2. Install MySQL.
 
+  .. note::
+    1. If your RHEL is a fresh install, you will need to add the [MySQL Yum repository](https://dev.mysql.com/doc/mysql-repo-excerpt/5.6/en/linux-installation-yum-repo.html).
+    2. You can download the release package (make sure you get the latest one) file with wget (wget "https://dev.mysql.com/get/mysql80-community-release-el8-1.noarch.rpm").
+    3. Once it's downloaded, you can install it with: `sudo yum localinstall platform-and-version-specific-package-name.rpm`
+    4. Next disable the system mysql: `sudo yum module disable mysql` and then install mysql
```

Comment:
```suggestion
    4. Next disable the system MySQL: `sudo yum module disable mysql` and then install MySQL.
```

Reply:
Question: Step 3 mentions installing the downloaded file. Is there another file that needs to be installed as per the second half of this step? Or is it already completed?

Reply:
In Step 2, the `rpm` file needs to be downloaded. For me it was called `mysql80-community-release-el8-1.noarch.rpm`. In step 3, the file downloaded in step 2 needs to be installed. So in other words:

platform-and-version-specific-package-name = mysql80-community-release-el8-1.noarch.rpm

I didn't know whether to use the exact filename, or leave it 'platform-and-version-specific', so I ended up choosing the worst option and using a combination of both. I agree this is confusing, so we should pick one, perhaps the exact filename? I'm fine with both.